pkgs/development/python-modules/pylibftdi/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, httpserver
+, libftdi1
+, libusb1
+}:
+
+buildPythonPackage rec {
+  pname = "pylibftdi";
+  version = "0.18.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "17c5h4xz1grynbpffngjflk3dlw2g2zbhkwb7h5v4n9rjdv41l5x";
+  };
+
+  propagatedBuildInputs = [
+    httpserver
+    libftdi1
+    libusb1
+  ];
+
+  postPatch = ''
+    substituteInPlace pylibftdi/driver.py \
+      --replace "self._load_library('libusb')" "'${libusb1.out}/lib/libusb1.so'" \
```

```suggestion
      --replace "self._load_library('libusb')" "self._load_library('${libusb1.out}/lib/libusb1.so')" \
```
and the same for the next line